src/SFA.DAS.AssessorService.Database/StoredProcedures/GetPreviousAddresses.sql

```diff
@@ -0,0 +1,30 @@
+﻿CREATE PROCEDURE [dbo].[GetPreviousAddresses]
+	-- Add the parameters for the stored procedure here
+	  @UserName nvarchar(30)	
+AS
+BEGIN	
+	declare @OrganisationId uniqueidentifier
+
+	SET NOCOUNT ON;
+
+	set @OrganisationId = (SELECT OrganisationId FROM Contacts 
+			WHERE UserName = @UserName)
+  
+	SELECT DISTINCT TOP 10 
+	c.OrganisationId
+    ,JSON_VALUE([CertificateData],'$.ContactAddLine1') AS AddressLine1
+   ,JSON_VALUE([CertificateData],'$.ContactAddLine2') AS AddressLine2
+   ,JSON_VALUE([CertificateData],'$.ContactAddLine3') AS AddressLine3
+   ,JSON_VALUE([CertificateData],'$.ContactAddLine4') AS City
+      ,JSON_VALUE([CertificateData],'$.ContactPostCode') AS PostCode
+	  ,MAX(c.CreatedAt) As CreatedAt
+	FROM [SFA.DAS.AssessorService].[dbo].[Certificates] c
```

Comment:
Could you remove the db qualifier from this line please? :)

Reply:
yep missed that one ta